e2e/scripts/run.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+set -eu
+set -o pipefail
+
+# Install Playbooks
+echo "Installing playbooks ..."
+docker exec \
+  ${CONTAINER_SERVER} \
+  sh -c "/mattermost/bin/mmctl --local plugin add /mattermost/prepackaged_plugins/mattermost-plugin-playbooks-*.tar.gz && /mattermost/bin/mmctl --local plugin enable playbooks"
+
+# Copy built plugin into server
+echo "Copying calls plugin into ${CONTAINER_SERVER} server container ..."
+docker cp dist/*.tar.gz ${CONTAINER_SERVER}:/mattermost/bin/calls
+
+# Install Calls
+echo "Installing calls ..."
+docker exec \
+  ${CONTAINER_SERVER} \
+  sh -c "/mattermost/bin/mmctl --local plugin add bin/calls"
+
+# Generates a sysadmin that Playwright can use
+echo "Generating sample data with mmctl ..."
+docker exec \
+  ${CONTAINER_SERVER} \
+  sh -c "/mattermost/bin/mmctl --local sampledata"
```

Comment:
Note to self: I think we just need the admin as the tests themselves will create any team/channel/user as needed so generating the whole sample data is just more work for not much benefit I think.